src/goto-unwind/src/goto_unwind_parse_options.cpp

```diff
@@ -102,6 +102,13 @@ struct goto_unwind_optionst
   }
 };
 
+// due to an interesting quirk with C++11 we do need to define these
+// even if they are static constexpr... C++17 removed this limitation
+// so if we ever manage to upgrade please remove these lines and forget
+// they ever existed.
+constexpr const char *goto_unwind_optionst::unwind_limit_option_name;
```

Comment:
:grey_question: Could we avoid the need for this by using `const` instead of `constexpr`? It isn't like this is performance sensitive or like we are adding static asserts based on it.

Reply:
@thomasspriggs it's more of a semantics thing for me, it’s a constexpr because it’s a constant expr. The only difference in making this `const` would be that this being required wouldn't be weird anymore, it’d still be required.